crucible.mjs

```diff
@@ -70,16 +70,72 @@ Hooks.once("init", async function() {
   /**
    * Configurable properties of the system which affect its behavior.
    */
+  function itemPacksByType(type) {
```

Comment:
One thing we need to be quite cautious of here is not to invalidate the setting value if a module is disabled. Suppose a user adds a custom pack to `ancestrySources` like `"myModule.myPack"` and then `myModule` gets disabled. In that event `"myModule.myPack"` will no longer be a valid choice for the setting and (I'm pretty sure) will cause some problems.

I think you need to test this case carefully and ensure that registering a pack here and then disabling the module that registered it doesn't break the system or cause the entire setting to get reset to default. 

We likely want the setting itself to be tolerant of any string value but present a multi-select element to the user which only includes (a) current values and (b) currently available pack options.

Make sense?

Reply:
Yeah, this would be my suggestion. Make the setting schema inherently tolerant, but:

1. Only present currently available packs in the UI selection
2. When setting the `CONFIG` values, make sure to only set the subset packs that currently exist.